src/languageTools/styles/default_provider_style.css

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2019 - present Adobe. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+
+span.brackets-hints-with-type-details {
+    width: 300px;
+    display: inline-block;
+}
+
+.brackets-hints-type-details {
+    color: #a3a3a3 !important;
+    font-weight: 100;
+    font-style: italic !important;
+    margin-right: 5px;
+    float: right;
+}
+
+.hint-description {
+    display: none;
+    color: #d4d4d4;
+    word-wrap: break-word;
+    white-space: normal;
+    box-sizing: border-box;
+}
+
+.hint-doc {
+    display: none;
+    padding-right: 10px !important;
+    color: grey;
+    word-wrap: break-word;
+    white-space: normal;
+    box-sizing: border-box;
+    float: left;
+    clear: left;
+    max-height: 2em;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    line-height: 1em;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+}
+
+.highlight .hint-description {
+    display: block;
+    color: #6495ed !important;
+}
+
+.highlight .hint-doc {
+    display: -webkit-box;
```

Comment:
Should this and `-webkit-box-orient` just be `display: flex` and `flex-direction: column`?

Reply:
See my comment above.